src/app/pages/common/entity/entity-form/entity-form.component.ts

```diff
@@ -557,10 +557,12 @@ export class EntityFormComponent implements OnInit, OnDestroy, OnChanges, AfterV
     return this.fb.control({disabled, value}, validation);
   }
 
-  setDisabled(name: string, disable: boolean, hide: boolean = false, status?:string) {
+  setDisabled(name: string, disable: boolean, hide?: boolean, status?:string) {
```

Comment:
I think you don't need to change the setDisabled function.

Reply:
Apparently this is how master already is, so merging it will make stable the same as master.